app/src/main/java/com/nextcloud/talk/ui/dialog/SetStatusDialogFragment.kt

```diff
@@ -177,17 +177,19 @@ class SetStatusDialogFragment :
                 override fun onNext(statusOverall: StatusOverall) {
                     if (statusOverall.ocs?.meta?.statusCode == HTTP_STATUS_CODE_OK) {
                         backupStatus = statusOverall.ocs?.data!!
```

Comment:
better use

statusOverall.ocs?.data?.let{
..... it.
}

in case data is null somehow